### DISCUSSION TITLE
Why can't some of my modules be placed in devDependencies?

### DISCUSSION THREAD

**Discussion by bbhxwl** [nuxt/nuxt#27798] [Questions]
### Environment

nuxt 3.12.2

### Reproduction

I don't use Prisma and it's okay to place it in devDependencies, but I can't build it if I use Prisma and place it in devDependencies. Why?

### Describe the bug

I don't use Prisma and it's okay to place it in devDependencies, but I can't build it if I use Prisma and place it in devDependencies. Why?

**Comment by TheAlexLichter** [comment]
It shouldn't matter for most of the cases if devDeps or deps ☺️

**Reply by bbhxwl** [reply]
But I'm having a very strange problem now. Have you ever used @prisma/nuxt?

**Reply by bbhxwl** [reply]
No one can solve many strange problems. Can only dev, cannot build.

**Reply by bbhxwl** [reply]
https://github.com/bbhxwl/testPrisma

**Comment by manpreet-compro** [comment]
@bbhxwl I see you have another open discussions also. Are all these related?
https://github.com/nuxt/nuxt/discussions/27786
https://github.com/nuxt/nuxt/discussions/27792

Also how did you create the nuxt project? via cli or diff way?

**Reply by bbhxwl** [reply]
> @bbhxwl I see you have another open discussions also. Are all these related? #27786 #27792
> 
> Also how did you create the nuxt project? via cli or diff way?

node 18.20.2 
nuxt 3.12.2 
cli ,
<img width="763" alt="image" src="https://github.com/nuxt/nuxt/assets/26525599/2ec06b58-03a8-4288-b05b-491ffbc18a44">
